src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -394,15 +394,10 @@ class EmojiPickerMenu extends Component {
             this.setFirstNonHeaderIndex(this.emojis);
             return;
         }
+        const newFilteredEmojiList = EmojiUtils.suggestEmojis(`:${normalizedSearchTerm}`, this.emojis.length);
 
-        // Skip "Frequently Used" emojis to avoid duplicate results.
-        // Frequently used emojis are on the 0th index category. Hence, slice the array from the 1st index category onwards.
-        const uniqueEmojis = _.isEmpty(this.props.frequentlyUsedEmojis) ? this.emojis : this.emojis.slice(this.unfilteredHeaderIndices[1] * this.numColumns);
-        const newFilteredEmojiList = _.filter(uniqueEmojis, emoji => (
-            !emoji.header
-            && !emoji.spacer
-            && _.find(emoji.keywords, keyword => keyword.includes(normalizedSearchTerm))
-        ));
+        // Sort the emojis by index so that we always get the same order of emojis
```

Comment:
Just curious ... what happens if we just don't do this sort?

Reply:
I'm also confused because this sort seems like it's explicitly not doing anything to change the sorting order. The items from the array returned by `EmojiUtils.suggestEmoji` [don't have an `index` field](https://github.com/Expensify/App/blob/9fff1af2ac383bd0832155ee9baf03a11424242a/src/libs/EmojiUtils.js#L231), which makes me think that the `.index` property is coming from `sort` itself. So this sort is a no-op right?

Reply:
Yep it's a no-op. I thought I had already gotten rid of it but I must not have committed that change. Updated!

Reply:
Oh, no it seems like it reverses the array:

<img width="468" alt="image" src="https://user-images.githubusercontent.com/47436092/212427503-6131d625-c621-480a-a416-79000cff0304.png">

If that's the case ... could we fix this by changing these lines;

- https://github.com/Expensify/App/blob/9fff1af2ac383bd0832155ee9baf03a11424242a/src/libs/EmojiUtils.js#L231
- https://github.com/Expensify/App/blob/9fff1af2ac383bd0832155ee9baf03a11424242a/src/libs/EmojiUtils.js#L239

To use `.push` instead of `.unshift`? I think that makes sense because, if we assume we encounter items in the `Trie` in order of "best match first", then it would make sense that when we encounter new items they get pushed to the back of the array instead of the front. That way the first item encountered in the Trie search (i.e: "best match") is at the start of the array, not the end.